rust/relay/src/server/client_message.rs

```diff
@@ -37,7 +38,20 @@ impl Decoder {
         // De-multiplex as per <https://www.rfc-editor.org/rfc/rfc8656#name-channels-2>.
         match input.first() {
             Some(0..=3) => {
-                let message = self.stun_message_decoder.decode_from_bytes(input)??;
+                let message = match self.stun_message_decoder.decode_from_bytes(input)? {
+                    Ok(message) => message,
+                    Err(broken_message) => {
+                        let method = broken_message.method();
+                        let transaction_id = broken_message.transaction_id();
+                        let error = broken_message.error().clone();
+
+                        tracing::debug!(transaction_id = %hex::encode(transaction_id.as_bytes()), method = %method.as_str(), %error, "Failed to decode attributes of message");
```

Comment:
Do we want to `warn!` on this instead so that we can see these in prod? Or might that open us up for abuse from unauthorized clients?

Reply:
We do see these in prod because we log on debug by default: https://github.com/firezone/firezone/blob/41b67b70826e9561f2863f27e5b3cbbc8f105bcf/terraform/environments/staging/main.tf#L690

Reply:
I would say a `warn` is a bit too excessive because the software is still completely functional, it is just bad input. More likely, this will be useful to discover bugs in a STUN-client or our STUN library when encoding / decoding attributes. I'd say `debug` is appropriate for that.

Reply:
We have metrics that record number of success and error responses. I'd suggest to set an alert on the number of error responses to spot potentially malicious behaviour.